crates/zed/src/languages/ocaml/config.toml

```diff
@@ -0,0 +1,17 @@
+name = "OCaml"
+path_suffixes = ["ml"]
+block_comment = ["(* ", "*)"]
+autoclose_before = ";,=)}]"
+brackets = [ 
+  { start = "{", end = "}", close = true, newline = true },
+  { start = "<", end = ">", close = true, newline = true },
+  { start = "[", end = "]", close = true, newline = true },
+  { start = "[|", end = "|", close = true, newline = true, not_in = ["string"] },
+  { start = "(", end = ")", close = true, newline = true },
+  { start = "\"", end = "\"", close = true, newline = false, not_in = ["string"] },
+  { start = "begin", end = "end", close = true, newline = true },
+  { start = "struct", end = "end", close = true, newline = true },
+  { start = "sig", end = "end", close = true, newline = true },
+  { start = "object", end = "end", close = true, newline = true },
```

Comment:
FYI: I was never able to get the `object` closed with `end` in the code, but was able to do that with `sig`

I also think all word-like "brackets" would benefit from some space in the start of the closing part, wdyt?
![image](https://github.com/zed-industries/zed/assets/2690773/0efd2843-95cb-4018-a229-f9aa6f9570d3)

Reply:
1. So i'm testing this now, and it seems like having start as "object" might be a reserved word as anything else e.g "objec" works fine. Not sure how to fix this.. A temporary hack could be to do it as "object "

2. Yes, that sounds great. I'll prepend whitespace before each "end".

Reply:
I've partially fixed 1 with "object " for now. It'll require you to type object with a space but it's better than nothing i guess.

done 2.

Reply:
Indeed, thanks for taking care of that.